webapp/tests/mocha/unit/initial-replication.spec.js

```diff
@@ -0,0 +1,968 @@
+const sinon = require('sinon');
+require('chai').use(require('chai-as-promised'));
+const rewire = require('rewire');
+const { expect } = require('chai');
+const initialReplication = rewire('../../../src/js/bootstrapper/initial-replication');
+const utils = require('../../../src/js/bootstrapper/utils');
+
+let localDb;
+let remoteDb;
+let userCtx;
+let clock;
+let onPromise;
+
+const FLAG_ID = '_local/initial-replication';
+
+/** global window **/
```

Comment:
I see this showing up more and more... It's an eslint flag, right? Why not update the eslint config instead?